internal/xds/translator/extproc.go

```diff
@@ -127,6 +130,9 @@ func extProcConfig(extProc ir.ExtProc) *extprocv3.ExternalProcessor {
 
 	if extProc.ResponseBodyProcessingMode != nil {
 		config.ProcessingMode.ResponseBodyMode = buildExtProcBodyProcessingMode(extProc.ResponseBodyProcessingMode)
+		if config.ProcessingMode.ResponseBodyMode == extprocv3.ProcessingMode_FULL_DUPLEX_STREAMED {
```

Comment:
should this be moved into buildExtProcBodyProcessingMode ?

Reply:
created a dedicated function for handling all the processing mode translation.